tests/lib/rules/line-comment-position.js

```diff
@@ -28,9 +28,6 @@ ruleTester.run("line-comment-position", rule, {
         "1 + 1; /* eslint-enable */",
         "1 + 1; // eslint-disable-line",
         "// eslint-disable-next-line\n1 + 1;",
-        "1 + 1; // global MY_GLOBAL, ANOTHER",
-        "1 + 1; // globals MY_GLOBAL: true",
-        "1 + 1; // exported MY_GLOBAL, ANOTHER",
```

Comment:
These lines should actually still be valid from the perspective of the `line-comment-position` rule but we now create a warning when parsing them. Is there a sensible way to keep these assertions? Maybe I should add them to the `invalid` section but somehow specify that we only expect them to error with the new error and not this rule's error?